checks/sast.go

```diff
@@ -16,28 +16,30 @@ package checks
 
 import (
 	"github.com/google/go-github/v32/github"
-	"github.com/ossf/scorecard/checker"
+	"github.com/ossf/scorecard/lib"
 )
 
+const sastStr = "SAST"
```

Comment:
In go `sastStr` suffix aren't standard. 

The same comment is applicable across other changes

Reply:
See comment above.

Reply:
Ack.